component/loki/source/file/decompresser.go

```diff
@@ -106,37 +117,35 @@ func newDecompressor(metrics *metrics, logger log.Logger, handler loki.EntryHand
 //
 // The selected reader implementation is based on the extension of the given file name.
 // It'll error if the extension isn't supported.
```

Comment:
This docstring is not accurate now, let's change this. Maybe mention what happens to files that cannot be decoded correctly?